misc/scripts/install-local.sh

```diff
@@ -110,6 +110,32 @@ function log() {
 	fi
 }
 
+
+function makeVirtualDeb {
+	# creates empty .deb package (with only the control file) for apt integration
+	# implements $(gives) variable
+	# Note: I only put "development" in "Section" because I had to put something there
+	sudo mkdir -p $SRCDIR/$name-pacstall/DEBIAN
+	echo "Package: $name-pacstall
+Version: $version
+Depends: ${depends//' '/' | '}
```

Comment:
Add optdepends to depends list if user decides to install them

Reply:
Also, since apt would install the dependencies, we don't need to explicitly install them.

Reply:
Also, add a check on cleanup if it installation failed to remove the deb

Reply:
I don't think I should put optdepends in depends but I'll put them in suggests so uninstalling them is also optional

Reply:
Ye, that works well for us